### ISSUE TITLE
Facebook videos?

### ISSUE THREAD

**Issue by nobodyspecial** [andy-portmen/open-in-vlc#64]
Facebook live streams don't work with this; are there any plans to support them?

**Comment by andy-portmen** [comment]
Right-click on the action button and use send page to VLC option. Does it work for you?
